### ISSUE TITLE
Bogus Unexpected '/'. ?

### ISSUE THREAD

**Issue by chrisjj** [douglascrockford/JSLint#224]
On https://www.jslint.com/ , this

```
var needlesRE = needles
.replace(/\S/g/*non-ws*/,function(c){
})
```

gives

![image](https://user-images.githubusercontent.com/6157073/38712117-8496ce72-3ec1-11e8-9bb9-d252f75e084c.png)

I see no invalid / there.

Also FWIW the output says "JSLint was unable to finish." and I see no reason why it couldn't finish.

**Comment by douglascrockford** [comment]
It is warning you about the visual confusion between regexp literals and comments.

**Comment by chrisjj** [comment]
> It is warning you about the visual confusion between regexp literals and comments.

Self-evidently not. It says "Unexpected". There's nothing _unexpected_ about a comment near a regex.

And even if there were, that's no excuse for this so-called warning in fact being a fatal error (which it is).

Workaround: Add a space before the comment
Suggested fix: Add "Tolerate [x] comments near regex", checkmarked by default.
